tests/drivers/gpio/gpio_get_direction/boards/nrf54l15bsim_nrf54l15_cpuapp.overlay

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "nrf52_bsim.overlay"
```

Comment:
Was not aware that overlays could include other overlays like this :)

Question is of course if it should. Looks a bit strange to me that the nrf54l15 cpuapp uses/includes the same overlay as the nrf52_bsim, especially when the `nrf52840dk_nrf52840.overlay` file doesn't

Reply:
Doing this saves a lot of duplicate code. It can feel a bit funny to reuse overlays between SOCs, but lukily this is only configuring the GPIOs and what we change is common between them.
(Wrt. this particular test there is no overlay today for the nrf52840dk.)